src/data/graphSamples.ts

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable */
+const samples: { [key: string]: any } = {
+  dense: {
+    graph: [
+      [0, 1, 2, 0, 0, 0, 0, 0, 0, 1],
+      [1, 0, 3, 4, 0, 0, 0, 0, 0, 0],
+      [2, 3, 0, 5, 6, 0, 0, 0, 0, 0],
+      [0, 4, 5, 0, 7, 8, 0, 2, 0, 0],
+      [0, 0, 6, 7, 0, 9, 0, 0, 0, 0],
+      [0, 0, 0, 8, 9, 0, 1, 3, 0, 0],
+      [0, 0, 0, 0, 0, 1, 0, 4, 1, 0],
+      [0, 0, 0, 2, 0, 3, 4, 0, 5, 1],
+      [0, 0, 0, 0, 0, 0, 1, 5, 0, 6],
+      [1, 0, 0, 0, 0, 0, 0, 1, 6, 0],
+    ],
+    string: '0 1 2 0 0 0 0 0 0 1\n1 0 3 4 0 0 0 0 0 0\n2 3 0 5 6 0 0 0 0 0\n0 4 5 0 7 8 0 2 0 0\n0 0 6 7 0 9 0 0 0 0\n0 0 0 8 9 0 1 3 0 0\n0 0 0 0 0 1 0 4 1 0\n0 0 0 2 0 3 4 0 5 1\n0 0 0 0 0 0 1 5 0 6\n1 0 0 0 0 0 0 1 6 0',
+  },
+  advanced: {
+    graph: [
+      [0, 2, 6, 0, 0, 0, 0],
+      [2, 0, 0, 5, 0, 0, 0],
+      [6, 0, 0, 8, 0, 0, 0],
+      [0, 5, 8, 0, 1, 5, 0],
+      [0, 0, 0, 1, 0, 6, 2],
+      [0, 0, 0, 5, 6, 0, 6],
+      [0, 0, 0, 0, 2, 6, 0],
+    ],
+    string: '0 2 6 0 0 0 0\n2 0 0 5 0 0 0\n6 0 0 8 0 0 0\n0 5 8 0 1 5 0\n0 0 0 1 0 6 2\n0 0 0 5 6 0 6\n0 0 0 0 2 6 0',
+  },
+  medium: {
+    graph: [
+      [0, 2, 6, 0, 4],
+      [2, 0, 0, 5, 0],
+      [6, 0, 0, 8, 1],
+      [0, 5, 8, 0, 9],
+      [4, 0, 1, 9, 0],
+    ],
+    string: '0 2 6 0 4\n2 0 0 5 0\n6 0 0 8 1\n0 5 8 0 9\n4 0 1 9 0',
+  },
+  neuron: {
+    graph: [
+      [0, 1, 2, 3, 4, 5, 6],
+      [1, 0, 0, 0, 0, 0, 0],
+      [2, 0, 0, 0, 0, 0, 0],
+      [3, 0, 0, 0, 0, 0, 0],
+      [4, 0, 0, 0, 0, 0, 0],
+      [5, 0, 0, 0, 0, 0, 0],
+      [6, 0, 0, 0, 0, 0, 0],
+    ],
+    string: '0 1 2 3 4 5 6\n1 0 0 0 0 0 0\n2 0 0 0 0 0 0\n3 0 0 0 0 0 0\n4 0 0 0 0 0 0\n5 0 0 0 0 0 0\n6 0 0 0 0 0 0',
```

Comment:
Now let's use `adjMatrixToGraphString` here

Reply:
wtf why we did not do that from the beginning

Reply:
`adjMatrixToGraphString` did not exist